src/metabase/models/dashboard_card.clj

```diff
@@ -42,19 +59,6 @@
     (apply set/union (mi/perms-objects-set card read-or-write) (for [series-card series]
                                                                  (mi/perms-objects-set series-card read-or-write)))))
 
-(defn- pre-insert [dashcard]
-  (let [defaults {:parameter_mappings     []
-                  :visualization_settings {}}]
-    (merge defaults dashcard)))
```

Comment:
Why is this thrown away?

Reply:
it's not, you can see the `define-before-insert` is doing the same thing.